<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recv_image</class>
 <widget class="QDialog" name="recv_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_emptyTemp">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>135</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Receive again</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SaveAs">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>135</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_error_correct">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>135</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Error Correct</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>135</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_recv_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
